src/script/common/c_content.cpp

```diff
@@ -47,6 +47,9 @@ struct EnumString es_TileAnimationType[] =
 	{0, nullptr},
 };
 
+static void push_engine_mask(lua_State *L, u16 engine_mask);
+static bool read_engine_mask(lua_State *L, int index, u16 &engine_mask);
```

Comment:
looks like my suggestion where to put `push_engine_mask` and `read_engine_mask` wasn't very good either 🙃

Reply:
I can move it back if you think it looks better before moving.

Reply:
doesn't matter